src/base/Context.ts

```diff
@@ -115,68 +116,46 @@ export class Context {
     }
   }
 
-  static async create(params: {
-    polymeshApi: ApiPromise;
-    middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-    seed: string;
-  }): Promise<Context>;
-
-  static async create(params: {
-    polymeshApi: ApiPromise;
-    middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-    keyring: CommonKeyring;
-  }): Promise<Context>;
-
-  static async create(params: {
-    polymeshApi: ApiPromise;
-    middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-    uri: string;
-  }): Promise<Context>;
-
-  static async create(params: {
-    polymeshApi: ApiPromise;
-    middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-    mnemonic: string;
-  }): Promise<Context>;
-
-  static async create(params: {
-    polymeshApi: ApiPromise;
-    middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-  }): Promise<Context>;
-
   /**
    * Create the Context instance
    */
   static async create(params: {
     polymeshApi: ApiPromise;
     middlewareApi: ApolloClient<NormalizedCacheObject> | null;
-    seed?: string;
-    keyring?: CommonKeyring;
-    uri?: string;
-    mnemonic?: string;
+    accountSeed?: string;
+    keyring?: CommonKeyring | UiKeyring;
+    accountUri?: string;
+    accountMnemonic?: string;
```

Comment:
Are these marked as breaking changes?

Reply:
They're not really breaking changes because the `Context` class isn't accessible externally

Reply:
Oh great, I thought it was the `connect` method